### PR TITLE
Extract UprobesTimerManager + basic event statistics

### PR DESCRIPTION
**Pull request by dpallotti**

### CODE REVIEW

File: OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp [hunk 1]
```diff
@@ -5,6 +5,119 @@
...
+  constexpr pid_t tid = 42;
+  bool process_uretprobes_res;
+  Timer processed_timer;
+  UprobesTimerManager timer_manager;
+
+  timer_manager.ProcessUprobes(tid, 1, 100);
+
+  process_uretprobes_res =
+      timer_manager.ProcessUretprobes(tid, 2, &processed_timer);
+  EXPECT_TRUE(process_uretprobes_res);
```
Comment (dimitry-) [inline OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp]: ASSERT_TRUE (if call failed there is no point in checking the rest)

File: OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp [hunk 2]
```diff
@@ -5,6 +5,119 @@
...
+  process_uretprobes_res =
+      timer_manager.ProcessUretprobes(tid2, 4, &processed_timer);
+  EXPECT_TRUE(process_uretprobes_res);
+  EXPECT_EQ(processed_timer.m_TID, tid2);
+  EXPECT_EQ(processed_timer.m_Start, 2);
+  EXPECT_EQ(processed_timer.m_End, 4);
+  EXPECT_EQ(processed_timer.m_Depth, 0);
+  EXPECT_EQ(processed_timer.m_FunctionAddress, 200);
+
+  timer_manager.ProcessUprobes(tid, 1, 100);
```
Comment (dimitry-) [inline OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp]: why is it here? you do not seem to check anything after this call

File: OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp (hunk 1 above)
Comment (dpallotti) [reply OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp]: Changed it.

File: OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp (hunk 2 above)
Comment (dpallotti) [reply OrbitCore/LinuxUprobesUnwindingVisitorTests.cpp]: My bad, removed it.

**Pull request merged**